src/sonic_ax_impl/mibs/ietf/rfc1213.py

```diff
@@ -153,6 +153,7 @@ class InterfacesUpdater(MIBUpdater):
     def __init__(self):
         super().__init__()
         self.db_conn = mibs.init_db()
+        self.multi_db_conn = mibs.init_multi_db() 
```

Comment:
>multi_db_conn [](start = 13, length = 13)

Please add unit tests to cover new features.

Reply:
Added for init_namespace_dbs(). Have to work on namespace specific SonicV2Connector/load_global_config(). Can this be done in a follow up PR?

Reply:
Updated unit tests for multiple namespace